app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -31,7 +31,7 @@
   }
 }
 
-.gem-c-input__suffix {
+.gem-c-input__prefix, .gem-c-input__suffix {
```

Comment:
@GDSNewt You don't need `border-left: 0;`for both prefix and suffix. It's only for the `.gem-c-input__suffix`.

It's also a good idea to but the classes in separate lines for readibility:

```
.gem-c-input__prefix,
.gem-c-input__suffix {

}
```